package.xml

```diff
@@ -74,53 +74,55 @@
     <license uri="https://github.com/DataDog/dd-trace-php/blob/master/LICENSE">BSD 3-Clause</license>
     <notes>
         <![CDATA[
-This release adds beta support for PHP Single Step Instrumentation, allowing the instrumentation
-of your PHP applications without the manual installation of the dd-trace-php extension.
-
-Read https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/single-step-apm/ for
-more details.
-
 ## Tracer
 
 ### Added
-- Add span field meta_struct #2673
-- Add config to disable obfuscation in memcached queries #2725
-- Add PDO spans to orphans removal #2743
-- Make generators replaceable #2718
-- Source Code Integration #2731
-- OpenAI Integration #2685
-- Add support for Single Step Instrumentation #2684 #2723
+- Add SpanEvents support for DD and OTEL #2754
+- Symfony Messenger Integration #2749
```

Comment:
```suggestion
- Symfony Messenger Integration #2749 (Thanks @natepage)
```

Or wherever you'd like, but we should mention it somewhere. Or maybe in the actual release notes?